src/command.rs

```diff
@@ -127,7 +127,8 @@ impl DockerCommandBuilder {
     }
 
     pub fn add_environment(mut self, var: &str, bind: &str) -> Self {
-        self.environment.push((var.to_string(), bind.to_string()));
+        self.environment.push("-e".into());
+        self.environment.push(format!("{}={}", var, bind));
```

Comment:
I think the only negative I can think of about this is that we might want to access these variables before building the environment for some reason. But it's unlikely to matter.

Reply:
Yeah, maybe. If needed we can move them back to the old format. I just saw two copies for one thing and thought it seemed unnecessary.